tests/unit/fem/test_assemblediagonalpa.cpp

```diff
@@ -21,7 +21,7 @@ int dimension;
 
 double coeffFunction(const Vector& x)
 {
-   if (dimension == 2)
```

Comment:
Getting rid of this `global` `dimension` is good: we can remove it and use local `for` int variable.

Reply:
Done in ffb9b753517a1aba51a571c9beb2cd475fe1a6d8